testsuite/features/step_definitions/salt_steps.rb

```diff
@@ -650,11 +650,13 @@ def pillar_get(key, minion)
 end
 
 When(/^I enable repositories before installing Salt on this "([^"]*)"$/) do |host|
-  step %(I enable repository "tools_additional_repo" on this "#{host}" without error control)
+  repo = $product == 'Uyuni' ? 'salt3002_repo_repo' : 'tools_additional_repo'
```

Comment:
not sure that salt3002_repo_repo would be there forever or it's an additional_repo

Reply:
you are right, it won't be here forever... it means we'll need to adapt this after a while :-(